core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -194,6 +194,24 @@ span.additional-metric-title {
   color: #777;
 }
 
+#additional-metrics {
+  float: left;
+  padding-bottom: 10px;
+}
+
+#links {
+  float: left;
+  padding-left: 100px;
```

Comment:
*shrug* using absolute layout and clearing float feels worse than tables, but, the page isn't exactly optimized anyway. It seems like the "jump to" links should be at the top and could just use inline layout like the rest of this, even if it bumps everything down a line. Or... if floating, how about at the top-right to use some of the whitespace?